packages/@atjson/renderer-react/test/react-renderer.test.tsx

```diff
@@ -379,102 +379,4 @@ describe("ReactRenderer", () => {
       );
     });
   });
-
-  describe("Subdocuments", () => {
```

Comment:
We don't need another test for the caption related changes? Looks like the other tests have caption related things?

Reply:
Subdocuments are deprecated in favor of slices and we haven't used them for many months at this point.